test_scripts/API/RAI/001_ATF_N_HMI_Does_Not_Respond_RAI_steeringWeelLocation.lua

```diff
@@ -0,0 +1,83 @@
+---------------------------------------------------------------------------------------------
```

Comment:
@istoimenova 
Please rename pull request with more descriptive name. It is hard to review batch of pull requests with the same names.